build-static.sh

```diff
@@ -330,8 +330,8 @@ CGO_ENABLED=1 \
 	${XCADDY_COMMAND} build \
 	--output "../dist/${bin}" \
 	${XCADDY_ARGS} \
-	--with github.com/dunglas/frankenphp=.. \
-	--with github.com/dunglas/frankenphp/caddy=.
+	--with frankenphp.dev=.. \
+	--with frankenphp.dev/caddy=.
```

Comment:
This strikes me as very strange, couldn't it point to `github.com/php/frankenphp` instead?

```console
XCADDY_ARGS="
    --with github.com/baldinof/caddy-supervisor
    --with github.com/caddyserver/cache-handler
    --with github.com/dunglas/caddy-cbrotli
    --with github.com/dunglas/mercure/caddy
    --with github.com/dunglas/vulcain/caddy
    --with frankenphp.dev
    --with frankenphp.dev/caddy"
    ```